### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ahiliitb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,112 @@
...
+**Note**: For *Google Colab* use '!' before every command.  
+
+
+## Training  
+### Classification  
+**Adding Data for Training**  
+- Create a 'data' folder within a 'content' folder.  
+- The 'data' folder will contain three different folders named 'train', 'test', and 'eval' used for training, testing, and evaluation purposes.  
+- Each of the 'train', 'test', and 'eval' folders contain data sets of different categories on which you want to use your model  
+- Data folder structure 'content'->'data'->('train', 'test', 'eval')->(category1, category2, category3, .....)
```
Comment (ishan121028) [inline README.md:21]: This can be replaced with a directory tree kind of representation.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,112 @@
...
+- Run the following code to train your model and you can change the default arguments with your custom arguments  
+
+```
+import UniTrain
+from UniTrain.utils.segmentation import get_data_loader, train_model, generate_model_summary
+from UniTrain.models.segmentation import UNet
+from UniTrain.utils.segmentation import parse_folder
+import torch
+import os
+import glob
```
Comment (ishan121028) [inline README.md]: Not needed right now

File: README.md [hunk 3]
```diff
@@ -1,2 +1,112 @@
...
+
+
+if parse_folder(data_dir):    
+    train_path = os.path.join(data_dir, "train")
+    test_path = os.path.join(data_dir, "test")
+    
+    train_image_paths = glob.glob(os.path.join(train_path, "images", "*.jpg"))
+    train_mask_paths = glob.glob(os.path.join(train_path, "masks", "*.png"))
+    
+    test_image_paths = glob.glob(os.path.join(test_path, "images", "*.jpg"))
```
Comment (ishan121028) [inline README.md]: not needed

File: README.md [hunk 4]
```diff
@@ -1,2 +1,112 @@
...
+    test_mask_paths = glob.glob(os.path.join(test_path, "masks", "*.png"))
+    
+    train_data_loader = get_data_loader(image_paths=train_image_paths, mask_paths=train_mask_paths, batch_size=32, shuffle=True, transform=None)
+    test_data_loader = get_data_loader(image_paths=test_image_paths, mask_paths=test_mask_paths, batch_size=32, shuffle=True, transform=None)
```
Comment (ishan121028) [inline README.md]: only a data_dir is passed in the new implementation

File: README.md [hunk 5]
```diff
@@ -1,2 +1,112 @@
...
+
+```
+import UniTrain
+from UniTrain.utils.DCGAN import parse_folder, get_data_loader, train_model
+from UniTrain.models.DCGAN import disc, gen
+import glob
+import torch
+
+if parse_folder('data'):
+    real_image_paths = glob.glob("data/real_images/images/*.jpg")
```
Comment (ishan121028) [inline README.md:92]: If this is how the DCGAN is implemented then we must change it, we shouldn't require users to care about external libraries such as glob for training.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,112 @@
...
+    real_image_paths = glob.glob("data/real_images/images/*.png")
+    real_image_paths = glob.glob("data/real_images/images/*.jpeg")
+    
+    train_dataloader = get_data_loader('data', 128 )
```
Comment (ishan121028) [inline README.md]: Add examples for batch size, shuffle, other arguments.

**Review by ishan121028 — CHANGES_REQUESTED** [review]

**Review by ishan121028 — APPROVED** [review]
